modules/nw-udn-cr.adoc

```diff
@@ -74,8 +74,8 @@ spec:
 +
 * The `subnets` field is mandatory.
 * The type for the `subnets` field is `cidr` and `hostSubnet`:
-** `cidr` is the cluster subnet and accepts a string value.
-** `hostSubnet` specifies the nodes subnet prefix that the cluster subnet is split to.
+** `cidr` specifies the supernet for your cluster. This parameter accepts a string value.
+** `hostSubnet` defines the subnet prefix.
```

Comment:
maybe  "defines the per-node subnet prefix" would give some more context

Reply:
Updated. Thanks, @asood-rh . So:

> `cidr` specifies the UDN for your cluster. This parameter accepts a string value.

Is more correct?